### PR TITLE
c64_cass.xml: Added 16 items (15 working, 1 not working)

### PR DESCRIPTION
**Pull request by scoomby**
New working software list additions
---------------------------------------
Willow Pattern (Firebird) [C64 Ultimate Tape Archive V2.0]
Winter Camp (Thalamus) [C64 Ultimate Tape Archive V2.0]
Winter Games (U.S. Gold) [C64 Ultimate Tape Archive V2.0]
Winter Olympiad 88 (Tynesoft) [C64 Ultimate Tape Archive V2.0]
Wiz (Melbourne House) [C64 Ultimate Tape Archive V2.0]
Wizardry (The Edge) [C64 Ultimate Tape Archive V2.0]
Wizball (Ocean) [C64 Ultimate Tape Archive V2.0]
Wonder Boy (Activision) [C64 Ultimate Tape Archive V2.0]
World Championship Soccer (Elite Systems) [C64 Ultimate Tape Archive V2.0]
World Class Leader Board: The Ultimate Golf Challenge (U.S. Gold) [C64 Ultimate Tape Archive V2.0]
World Cup Carnival (U.S. Gold) [C64 Ultimate Tape Archive V2.0]
World Games (U.S. Gold) [C64 Ultimate Tape Archive V2.0]
World Series Baseball (Imagine) [C64 Ultimate Tape Archive V2.0]
The Worm in Paradise (Level 9 Computing) [C64 Ultimate Tape Archive V2.0]
X-15 Alpha Mission (Activision) [C64 Ultimate Tape Archive V2.0]

New NOT_WORKING software list additions
---------------------------------------
Wimbledon '64 (Merlin Software) [C64 Ultimate Tape Archive V2.0]

### CODE REVIEW

File: hash/c64_cass.xml [hunk 1]
```diff
@@ -16305,6 +16523,42 @@ Some are based on the C64 Tape Archive at https://archive.org/download/c64tapes/
...
+	</software>
+
+	<software name="thewormi">
+		<description>The Worm in Paradise</description>
+		<year>1985</year>
+		<publisher>Level 9 Computing</publisher>
+
+		<part name="cass1" interface="cbm_cass">
+			<dataarea name="cass" size="465493">
+				<rom name="Worm_in_Paradise,_The.tap" size="465493" crc="4f3cc7b9" sha1="a8e027a8d8ff350816312eaf8adceec9f2fc2f3f"/>
```
Comment (cuavas) [inline hash/c64_cass.xml:16545]: Please don’t move articles to the end like this in tape labels.

Comment (scoomby) [reply hash/c64_cass.xml:16545]: The tape labels shown here are identical to the ones used in the C64 Ultimate Tape Archive V2.0.  I think I was asked not to change them in an earlier PR.

Comment (cuavas) [reply hash/c64_cass.xml:16545]: Yeah, the trouble is, these other projects tend to mess with the labels in all sorts of ways, while MAME is supposed to preserve stuff.  I guess we put it down as something to fix later when giving the software list an overhaul.

Comment (scoomby) [reply hash/c64_cass.xml:16545]: Agreed.

**Pull request merged**